### ISSUE TITLE
Scala interpolation

### ISSUE THREAD

**Issue by jedesah** [tpolecat/tut#34]
I would like to have String interpolation in tut template files. I am aware that this is somewhat of a separate concern, and would like to open up the discussion to alternate solutions.

I have though about providing an alternative plugin that can integrate with sbt-site to interpolate files with scala code. Then, however, one would have to modify sbt-site to be able to chain text transformation plugins, since I have a use case where I both want to have tut examples and String interpolation within the same file. This seems reasonable and would be a modular design although there might be some concerns around clashing between plugins and ordering.

In the meantime it might be reasonable to implement the feature directly in tut itself.

Thoughts?

**Comment by tpolecat** [comment]
Yeah, I have thought about this. Could get complex pretty quickly but I'll think about it some more ;-)

**Comment by aishfenton** [comment]
+1 for this feature. 

In my case I'm wanting to embed HTML output from Scala directly into the Markdown, since I'm trying to produce documentation like the following (except mine's a Scala wrapper to Vega-Lite). 
https://rud.is/projects/vegalite01.html

**Comment by magicknot** [comment]
Any news on this?

**Comment by tpolecat** [comment]
@magicknot I will probably do something like #104 for the next release ... soonish, but no promises.
